api/base/exceptions.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.exceptions import APIException
+
+#Custom Exceptions the Django Rest Framework does not support
+
+class Gone(APIException):
+    status_code = 410
```

Comment:
The exception defined in DRF use the `rest_framework.status` for these constants. For example:

```python
class NotFound(APIException):
    status_code = status.HTTP_404_NOT_FOUND
    default_detail = _('Not found.')
```

Instead of using the integer `410`, let's use `status.HTTP_410_GONE`, for consistency.